packages/components/src/Chip/Chip.tsx

```diff
@@ -48,7 +48,7 @@ export const Chip = ({
       <Typography size="base" fontWeight="medium">
         {heading}
       </Typography>
-      {heading && <span className={styles.chipBar} />}
+      {heading && label && <span className={styles.chipBar} />}
```

Comment:
I walked through this change with Chris and Scott - only show the chipBar when there is a header and label

Reply:
🙏